### DISCUSSION TITLE
Support for linux/arm64 images

### DISCUSSION THREAD

**Discussion by JanBerktold** [multani/docker-nomad#38] [Ideas]
We have been using this image for our local test environments very successfully -- thank you! Some engineers recently switched to M1/M2 Macbooks on which we're forced to use emulation since this image does not support linux/arm64 -- this comes with a signficiant performance penalty.

It would be great for this image to be published for several CPU architectures. Given that Nomad supports linux/arm64 natively (see https://www.nomadproject.io/downloads), it seems "relatively" straightforward to implement this using docker's buildx support.

Would you be open to accepting such a commit?

**Comment by multani** [comment]
Hey @JanBerktold!

> We have been using this image for our local test environments very successfully -- thank you!

I'm very glad it's useful to someone! Thanks for reporting this :)

> It would be great for this image to be published for several CPU architectures. Given that Nomad supports linux/arm64 natively (see https://www.nomadproject.io/downloads), it seems "relatively" straightforward to implement this using docker's buildx support.
> 
> Would you be open to accepting such a commit?

It would be a great addition and I would totally support this!
I haven't really looked into building arm64 Docker images generally so far, but if you are willing to contribute something, it would be really appreciated!

**Reply by multani** [reply]
Hey @JanBerktold!

I can give it a shot if you want, there are new versions to release soon and it would be great if they ship on other platforms as well!

**Reply by multani** [reply]
I gave it a shot there! https://github.com/multani/docker-nomad/pull/55

**Comment by multani** [comment]
Latest versions are built using Debian and support armv7 and arm64 :tada: , see:

* https://github.com/multani/docker-nomad/releases/tag/1.7.0-beta.1
* https://github.com/multani/docker-nomad/releases/tag/1.6.3
* https://github.com/multani/docker-nomad/releases/tag/1.5.10
* https://github.com/multani/docker-nomad/releases/tag/1.4.14
